feat: Add script functionality to transformer (#116)
grafana/killercoda: tempo/quick-start/intro.md

This quick start guide uses the `local`{{copy}} example running Tempo as a single binary (monolithic). Any data is stored locally in the `tempo-data`{{copy}} folder.
To learn more, read the [local storage example README](https://github.com/grafana/tempo/blob/main/example/docker-compose/local).

> > **Note**: A setup script is running in the background to install the necessary dependencies. This should take no longer than 30 seconds. Your instance will be ready to use once you `Setup complete. You may now begin the tutorial`.

Reviewer: ```suggestion
> **Note**:
> A setup script is running to install the necessary dependencies.
> This should take no longer than 30 seconds. 
> Your instance will be ready to use after you see `Setup complete. You may now begin the tutorial`.
```
